src/server/symbols.map

```diff
@@ -392,7 +401,21 @@ global:
     mir::input::VirtualInputDevice::if_started_then*;
     mir::input::default_cursor_size*;
     mir::input::input_platform_from_graphics_module*;
+    mir::input::make_unique_context*;
     mir::input::probe_input_platforms*;
+    mir::input::receiver::XKBMapperRegistrar::XKBMapperRegistrar*;
```

Comment:
Huh? Why are these in the 2.17 stanza now?

Reply:
Is this the fault of the tool, or did you do this on  purpose?

Reply:
Huh!? Everything should have been in the 2.18 stanza. Mo...

Reply:
Good catch. Fixed